### PR TITLE
Updated PA API routes to use googleHelpers (CU-8679128d6)

### PR DESCRIPTION
**Pull request by micahdbak**
This PR updates the PA API routes `/pa/aws-device-registration` and `/pa/buttons-twilio-number` to use `googleHelpers.paAuthorize` instead of `clickUpHelpers.clickUpChecker`.
It also upgrades `brave-alert-lib` to v10.3.1, which removes the `clickUpHelpers` export, with `googleHelpers` being its replacement.

Test plan:
- [x] Passes existing test cases
- [x] PA API routes work as expected with `particle-accelerator` updates

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-8679128d6 Handle Google authentication in BraveButtons server for PA API](https://app.clickup.com/t/8679128d6)

**Review by tmackinnon — APPROVED** [review]
Looks good to me - approved!

File: server/.env.example [hunk 1]
```diff
@@ -145,3 +145,13 @@ PA_API_KEY_PRIMARY=MyLongKey
 PA_API_KEY_PRIMARY_TEST=MyLongKey
 PA_API_KEY_SECONDARY=MyLongKeyBackup
 PA_API_KEY_SECONDARY_TEST=MyLongKeyBackup
+
+# The client ID of PA (for use in brave-alert-lib)
+# Get this from Google Cloud --> Brave PA Sign-In --> APIs & Services --> Credentials --> Web ID / Web ID (dev)
+PA_CLIENT_ID=fakeclientid.apps.googleusercontent.com
```
Comment (ahmaddaadaa) [inline server/.env.example:151]: would this need to be replaced with a real client ID before deployment? or Google API console would automatically provide a unique client ID ?

**Review by ahmaddaadaa — COMMENTED** [review]
Nice work Mika. Overall looks good! I left you one comment on the Client ID.

File: server/.env.example (hunk 1 above)
Comment (micahdbak) [reply server/.env.example:151]: Hi Ahmad, this environment variable needs to be set before deployment. The line `Get this from Google Cloud --> Brave PA Sign-In --> APIs & Services --> Credentials --> Web ID / Web ID (dev)` shows where you can find the PA client ID in the Google cloud console.

Comment (ahmaddaadaa) [reply server/.env.example:151]: Okay. Sound good. Thanks for the clarification.

**Review by ahmaddaadaa — APPROVED** [review]

**Pull request merged**